code/modules/antagonists/cult/team_cult.dm

```diff
@@ -0,0 +1,485 @@
+/datum/team/cult
+	name = "Cult"
+	antag_datum_type = /datum/antagonist/cultist
+
+	/// Does the cult have glowing eyes
+	var/cult_risen = FALSE
+	/// Does the cult have halos
+	var/cult_ascendant = FALSE
+	/// How many crew need to be converted to rise
+	var/rise_number
+	/// How many crew need to be converted to ascend
+	var/ascend_number
+	/// Used for the CentComm announcement at ascension
+	var/ascend_percent
+	/// Variable used for tracking the progress of the cult's sacrifices & god summonings
+	var/cult_status = NARSIE_IS_ASLEEP
+
+	/// God summon objective added when ready_to_summon() is called
+	var/datum/objective/eldergod/obj_summon
+	var/sacrifices_done = 0
+	var/sacrifices_required = 2
+
+	/// Are cultist mirror shields active yet?
+	var/mirror_shields_active = FALSE
+
+	// Is this for adminbus or debugging coderman, you decide.
+	var/no_announcements = FALSE
```

Comment:
???

Reply:
I mostly wanted a way for admins to disable the announcements if its needed, ill update the comment